src/plugins/discover/public/application/main/components/sidebar/discover_field.tsx

```diff
@@ -439,6 +442,21 @@ function DiscoverFieldComponent({
           originatingApp={PLUGIN_ID}
           uiActions={getUiActions()}
         />
+
+        <EuiSpacer size="s" />
+
+        {/* this should probably be inside FieldPopoverVisualize so it is placed inside the footer.
+        currently this places it below the footer  */}
+        <FieldPopoverCategorize
+          field={field}
+          dataView={dataView}
+          trackUiMetric={trackUiMetric}
+          contextualFields={contextualFields}
+          originatingApp={PLUGIN_ID}
+          uiActions={getUiActions()}
+          closePopover={() => closePopover()}
+          onAddDSLFilter={onAddDSLFilter}
+        />
```

Comment:
dear @jughosta , can you have a look at this, once back from PTO, many thx 🙏

Reply:
@kertal I think "Categorize" button could be moved inside Unified Field List if a new action is introduced and the logic of creating a filter (as in `onAddDSLFilter`) can also be encapsulated inside the button.